azkaban-common/src/main/java/azkaban/project/JdbcProjectLoader.java

```diff
@@ -402,9 +421,12 @@ private void uploadProjectFile(Connection connection, Project project,
           logger.info("Running update for " + filename + " chunk " + chunk);
           runner.update(connection, INSERT_PROJECT_FILES, project.getId(),
               version, chunk, size, buf);
+
+          // We enforce az committing to db when uploading every a single chunk.
```

Comment:
Explain the reason. 
How about " to reduce the transaction duration and conserve sql server resources"?